src/Avalonia.Dialogs/ManagedFileChooserSources.cs

```
@@ -71,6 +71,10 @@ public static ManagedFileChooserNavigationItem[] DefaultGetFileSystemRoots()
                        {
                            return null;
                        }
+                       catch (DirectoryNotFoundException _)
```

I think we should just ignore all exceptions here, since we aren't interested in reasons why we've failed to enumerate files and it's definitely not worth a crash

---

make sense. making the change